README.md

```diff
@@ -1 +1,171 @@
-# To-Do-list-best-practices
\ No newline at end of file
+ <a name="readme-top"></a>
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [To-Do-List] <a name="about-project"></a>
+
+**[To-Do-List]** To-do list  is is a JavaScript application that implements a to-do list created to help you orginize your daily activities by keeping them listed that you can consult whenever you want, with a feature of add, remove, and mark tasks as completed.
+It uses the DOM API to interact with the HTML elements, and it uses local storage to save the tasks.
+
+## 🛠 Built With <a name="built-with"></a>
+- **[Html]**
+- **[Css]**
+- **[JavaScript]**
+- **[webpack]**
+- **[npm]**
+  
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[list-structure]**
+- **[Adding_a_new_item]**
+- **[Removing_a_selected_item]**
+- **[Marking_a_selected_item_as_complete]**
+- **[Removing_all_items_marked_as_complete_at_once]**
+- **[Reordering_a_selected_item]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Click Here]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
```

Comment:
- if you have already published your project it's better to include the link. and of not just include a text `not published`.

Reply:
I have implemented your suggestions.